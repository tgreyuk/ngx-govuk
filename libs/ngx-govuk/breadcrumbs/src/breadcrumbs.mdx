import { Meta, Story, ArgsTable, Canvas } from '@storybook/blocks';
import * as stories from './breadcrumbs.component.stories';

<Meta of={stories} />

# Accordion

Provide contextual feedback messages for typical user actions with the handful of available and flexible alert messages.

## Usage

```ts
import { WdxAlertModule } from '@wdx/components/alert';`
```

## Selector

`<wdx-alert>`
