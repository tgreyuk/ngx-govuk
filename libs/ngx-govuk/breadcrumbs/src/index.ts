export * from './breadcrumb-item.directive';
export * from './breadcrumbs.component';
