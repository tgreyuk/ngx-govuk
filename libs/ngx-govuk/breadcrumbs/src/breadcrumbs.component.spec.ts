import { ComponentFixture, TestBed } from '@angular/core/testing';
import { NgxGovukBreadcrumbsComponent } from './breadcrumbs.component';

describe('NgxGovukBreadcrumbsComponent', () => {
  let component: NgxGovukBreadcrumbsComponent;
  let fixture: ComponentFixture<NgxGovukBreadcrumbsComponent>;

  beforeEach(async () => {
    await TestBed.configureTestingModule({
      imports: [NgxGovukBreadcrumbsComponent],
    }).compileComponents();

    fixture = TestBed.createComponent(NgxGovukBreadcrumbsComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('should create', () => {
    expect(component).toBeTruthy();
  });
});
