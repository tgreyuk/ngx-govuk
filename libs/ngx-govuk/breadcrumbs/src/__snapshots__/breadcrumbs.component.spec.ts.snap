// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NgxGovukBreadcrumbs should render default state 1`] = `
<ngx-govuk-breadcrumbs>
  <div
    class="govuk-breadcrumbs"
  >
    <ol
      class="govuk-breadcrumbs__list"
    >
      <li
        class="govuk-breadcrumbs__list-item"
      >
        <a
          href="#"
          ngxgovukbreadcrumbsitem=""
        >
          Home
        </a>
      </li>
      <li
        class="govuk-breadcrumbs__list-item"
      >
        <a
          href="#"
          ngxgovukbreadcrumbsitem=""
        >
          Travel abroad
        </a>
      </li>
    </ol>
  </div>
</ngx-govuk-breadcrumbs>
`;
